src/main.rs

```diff
@@ -19,10 +19,9 @@ mod check;
 mod template_files;
 mod tui;
 
-static TEMPLATE: LazyLock<Template> = LazyLock::new(|| {
-    let options = include_str!("../template/template.yaml");
-    serde_yaml::from_str(options).unwrap()
-});
+const OPTIONS: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/template/template.toml");
```

Comment:
it still needs `include_str!` - now we are parsing the path as YAML

Reply:
Still not working sadly :(